vivisect/symboliks/tests/test_reduce.py

```diff
@@ -131,7 +131,7 @@ def test_symboliks_reduce_funcargs_multipass(self):
         expr = Call(Const(0x400, 8), Const(8, 8), argsyms=[arg,])
 
         expr = expr.reduce()
-        self.assertEqual(str(expr), '1024((mem[0x00014000:8] ^ 0))')
```

Comment:
interesting...

Reply:
I changed foo=True so that reduction actually full reduces in all cases. It's been bothering me that we have these edge cases.

Reply:
i like it.